docs/site/installation.md

```diff
@@ -29,72 +28,77 @@ This short manual helps to install the SCION Microfrontend Platform and describe
+
+     As with micro applications, the host can provide a manifest to contribute behavior, as following:
+
+     ```ts
+     await MicrofrontendPlatform.startHost({
+       host: {
+         manifest: {
+           name: 'Host Application',
+           capabilities: [
+             // capabilities of the host application
+           ],
+           intentions: [
+             // intentions of the host application
+           ],
+         }
+       },
+       applications: [
+         {symbolicName: 'products-app', manifestUrl: 'http://localhost:4201/manifest.json'},
+         {symbolicName: 'shopping-cart-app', manifestUrl: 'http://localhost:4202/manifest.json'},
+       ],
+     });
+     ```
+
+     The method for starting the platform host returns a Promise that resolves once platform startup completed. You should wait for the Promise to resolve before interacting with the platform.
+   </details>
+
    <details>
-     <summary><strong>Start the platform in micro applications</strong></summary>
+     <summary><strong>Connect to the platform host in a micro application</strong></summary>
      <br>
-     
-   For a micro application to connect to the platform host, it must be registered in the host application. For this, the micro application must provide a manifest file.  
- 
-   3.1. *Providing a manifest*
-  
-      Create the manifest file, for example, `manifest.json`. The manifest declares at least the name of the application.
- 
-      ```json
-      {
-        "name": "Products App"
-      }
-      ```
-   
-   3.2. *Connecting to the platform host*
-   
-      ```ts
-      await MicrofrontendPlatform.connectToHost({symbolicName: 'products-app'});
-      ```
-   
-      As the symbolic name, you must pass the exact same name under which you registered the micro application in the host application.
-      
-      The method for connecting to the platform host returns a Promise that resolves when connected to the platform host, or that rejects if not finding the platform host or if the micro application is not authorized to connect. You should wait for the Promise to resolve before interacting with the platform.
-  
+
+     For a micro application to connect to the platform host, it needs to provide a manifest file and be registered in the host application.
+
+     Create the manifest file, for example, `manifest.json`. The manifest declares at minimum the name of the application.
+
+     ```json
+     {
+       "name": "Products Application"
+     }
+     ```
+
+     A micro application connects to the platform host by invoking the method `MicrofrontendPlatform.connectToHost` and passing its identity as argument. The host does check whether the connecting micro application is qualified to connect, i.e., is registered in the host application under that origin; otherwise, the host will reject the connection attempt.
```

Comment:
`The host checks` or `The host performs a check`

Reply:
Fixed